services/sts/src/main/java/software/amazon/awssdk/services/sts/auth/StsCredentialsProvider.java

```diff
@@ -40,6 +42,10 @@
 @ThreadSafe
 @SdkInternalApi
 abstract class StsCredentialsProvider implements AwsCredentialsProvider, SdkAutoCloseable {
+
+    private static Duration DEFAULT_STALE_TIME = Duration.ofMinutes(1);
```

Comment:
Can we change this to : ```private static final Duration DEFAULT_STALE_TIME = Duration.ofMinutes(1);```

Reply:
No worries. In my head it was final. Pushed